pygments/styles/staroffice.py

```diff
@@ -0,0 +1,28 @@
+"""
+    pygments.styles.staroffice
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Style similar to StarOffice style, also OpenOffice and LibreOffice.
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.style import Style
+from pygments.token import Keyword, Name, Comment, String, Error, \
+     Number, Operator, Generic
+
+class StarofficeStyle(Style):
+    """
+    Style similar to StarOffice style, also OpenOffice and LibreOffice.
+    """
+    default_style = ""
+    styles = {
+        Comment:                '#808080',   # Gray
+        Error:                  '#800000',   # Red
+        Keyword:                '#000080',   # Blue
+        Name:                   '#008000',   # Green
+        Number:                 '#FF0000',   # Lightred

```

Comment:
That's not "light" red, is it? The `#800000` marked "red" above is lighter ... Did you mix them up?

Reply:
Colors are correct, they match [Color Types](https://github.com/LibreOffice/core/blob/e4a57dcdabc9ae7d381025e008b90635c1b7b10c/include/tools/color.hxx#L453-L466)